File: infostar_cstm_app/docevents/item.py
# import frappe


# def update_conversion_factor(doc,event):
# 	if doc.attributes and doc.variant_of:
# 		for row in doc.attributes:
# 			if row.get('attribute') == 'Height':
# 				doc.height = int(row.get('attribute_value'))
# 			if row.get('attribute') == 'Width':
# 				doc.width = int(row.get('attribute_value'))
# 			if row.get('attribute') == 'Yield':
# 				doc.yield_ == float(row.get('attribute_value'))

# 	if not doc.has_variants:
# 		update_uoms_conversion(doc)


# def update_uoms_conversion(doc):
# 	for row in doc.uoms:
# 		if row in doc.uoms:
# 			if row.formula:
# 				row.conversion_factor = eval(str(row.formula))

File: infostar_cstm_app/public/js/item.js
frappe.ui.form.on("Item",{
	onload : function(frm){
		if (frm.doc.__islocal){
			auto_populate_fields(frm)
		}
	},
});




function auto_populate_fields(frm){

	var childTableRows = cur_frm.doc.attributes || [];
	for (var i = 0; i < childTableRows.length; i++){
		var row = childTableRows[i];
		if (row.attribute == 'Height'){
				cur_frm.doc.height = row.attribute_value
		}
		if (row.attribute == 'Width'){
			cur_frm.doc.width = row.attribute_value
		}
		if (row.attribute == 'Yield'){
			cur_frm.doc.yield = row.attribute_value
		}
		frm.refresh_fields();
	}
}

